src/lib/index.ts

```diff
@@ -405,10 +417,18 @@ export class SourceFileConverter {
           } else {
             commands.push(new ast.RawText(`<${node.tagName}`));
           }
-          for (const {name, value} of node.attrs) {
-            const textLiterals = value.split(markerRegex);
-            if (
-              name.startsWith('.') ||
+          for (let {name, value} of node.attrs) {
+            if (name.startsWith('.')) {
+              // Need to get name from source to ensure proper casing.
+              const propertyName = lastAttributeNameRegex.exec(strings[partTypes.length])![2];
+              const reflectedName = reflectedAttributeName(propertyName.slice(1), node.tagName);
```

Comment:
Is `reflectedName` the reflected attribute name? If so, add "attribute" in the name.